app/models/local_restriction.rb

```diff
@@ -0,0 +1,43 @@
+class LocalRestriction
+  FILE_PATH = "app/lib/local_restrictions/local-restrictions.yaml".freeze
+
+  attr_reader :gss_code
+
+  def initialize(gss_code)
+    @gss_code = gss_code
+  end
+
+  def area_name
+    area_data["name"]
+  end
+
+  def alert_level
+    area_data["alert_level"]
+  end
+
+  def guidance
+    area_data["guidance"]
+  end
+
+  def extra_restrictions
+    area_data["extra_restrictions"]
+  end
+
+  def start_date
+    area_data["start_date"].to_date
+  end
+
+  def end_date
+    area_data["end_date"].to_date
+  end
+
+private
+
+  def restriction_data
+    @restriction_data ||= YAML.load_file(FILE_PATH)
+  end
+
+  def area_data
```

Comment:
I wonder if we might need to make this public later, so that we can check if any restrictions exist for a gss code. i.e. `LocalRestriction.new("E000000001").area_data.present?` 🤔

Reply:
Yeah, good point. I guess we could use a proxy like `LocalRestriction.new("E000000001").area_name.present?` but it's not as clear.